lib/beaker/testmode_switcher/beaker_runners.rb

```diff
@@ -34,7 +41,7 @@ def resource(type, name, opts = {})
           end
         end
 
-        on(default,
+        on(default, # rubocop:disable Metrics/MultilineMethodCallBraceLayout
```

Comment:
Please just disable that rule globally, or configure it to use the proper style (according to http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MultilineMethodCallBraceLayout , `new_line` ?)